server/test/parties.test.js

```diff
@@ -188,3 +188,16 @@ describe('Parties', () => {
       });
   });
 });
+
+describe('Root Route', () => {
```

Comment:
This test does not have anything to do with parties. Why do we have it here?

Reply:
Its a test for the root route that covers both offices and parties. I had put it in parties.test.js earlier.